pyrk/inp/sim_info.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
         self.rho_ext = self.init_rho_ext(rho_ext)
         self.feedback = feedback
         self.ne = self.init_ne()
-        self.th = th_system.THSystem(kappa=kappa, components=components)
+        self.th = th_system.THSystemSphFVM(kappa=kappa, components=components)
```

Comment:
This makes all simulations use the SphFVM version of the THSystem class. This can't be the right strategy. Let's talk about why this is necessary and how to fix the design to make this make sense.